.github/workflows/dependabot-create-issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Create issue for dependabot pr
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened]
+
+
+jobs:
+    create_issue:
+        runs-on: ubuntu-latest
+        env:
+            GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        steps:
+          - uses: actions/checkout@v2
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+
+          - name: Open issue if Dependabot PR
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+            env:
+              pr_title: ${{github.event.pull_request.title}}
+              pr_number: ${{github.event.pull_request.number}}
+              pr_url: ${{github.event.pull_request.url}}
+            run: |
+              project="Data Catalogue"
+              label="Dependabot Issue"
+              title="Dependabot PR $pr_title opened"
+              body="Dependabot has opened PR #$pr_number url - $pr_url
```

Comment:
I think github might collapse #pr_number and #pr_url to the same thing, so you might not need both?

Reply:
Looking it [this](https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request) they still seem to be different things -

Reply:
I mean when it renders the text, I can type `#802` or `https://github.com/ministryofjustice/find-moj-data/pull/802` and they both get rendered as the same link, #802